settings.gradle

```diff
@@ -0,0 +1 @@
+include "benchmark"
```

Comment:
This registers the "benchmark" project as a sub-project of gax-java. They'll live in the same repo but otherwise separate.